pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -186,6 +159,46 @@ func (ac *Config) selectNode(nodeBlockDeviceMap map[string]*blockDeviceList) *no
 	// minRequiredDiskCount will hold the required number of disk that should be selected from a qualified
 	// node for specific pool type
 	minRequiredDiskCount := spcv1alpha1.DefaultDiskCount[ac.poolType()]
+
+	if ProvisioningType(ac.Spc) == ProvisioningTypeAuto {
```

Comment:
here as well, a comment on why this 'if' check required would be good

Reply:
can this be made as function?

Reply:
Yes, I will update it